auth/authenticators/no-auth-authenticator.ts

```diff
@@ -15,21 +15,20 @@
  */
 
 import { Authenticator } from './authenticator';
-import { AuthenticateOptions, AuthenticatorInterface } from './authenticator-interface';
+import { AuthenticateOptions } from './authenticator-interface';
 
-export class NoAuthAuthenticator extends Authenticator implements AuthenticatorInterface {
-  /**
-   * NoAuth Authenticator Class
-   *
-   * Provides a way to use a service without specifying credentials.
-   *
-   * @constructor
-   */
-  constructor() {    
+/**
+ * NoAuth Authenticator Class
+ *
+ * Provides a way to use a service without specifying credentials.
```

Comment:
I didn't comment on this during my first review, but after thinking about it a little more, this comment might cause ultra-security-conscious folks to have a coronary :)
Perhaps something like this would avoid a medical emergency for those folks:
```
The NoAuthAuthenticator is a placeholder authenticator implementation which performs no authentication of outgoing REST API requests.   It might be useful during development and testing.
```

Reply:
👍